### ISSUE TITLE
12400 / b660 get such a result.

### ISSUE THREAD

**Issue by duckjpsgaintas** [rcmaehl/WhyNotWin11#693]
![image](https://user-images.githubusercontent.com/19954379/183554169-5cda060b-7af0-41f0-b591-02744aadce83.png)

Also can't get a pass from the win 11 installer.

Thanks.

**Comment by micwoj92** [comment]
https://docs.microsoft.com/en-us/windows-hardware/design/minimum/supported/windows-11-supported-intel-processors

12400 is not on that list.

Try to download pc health check. Install it and run it should fix assistant.

**Comment by WilloIzCitron** [comment]
those are Alder Lake Family but it maybe Microsoft didn't put at list anyway, the list kinda outdated

**Comment by micwoj92** [comment]
@duckjpsgaintas Could you try 2.5.0.3?

**Comment by duckjpsgaintas** [comment]
I successfully installed win11 with the image made by the official
downloaded installation disk making software. Obviously there is a problem
with the image file I got earlier. But the whynotwin11 test still failed
after I finished the installation and tried it again on the installed win11
system. I already tried different versions. All failed.

If someone encounters problems like I did, it is recommended to go directly
to the official website to download the installation disk making program to
make the installation image/USB device for installation.

Thank you all for helping me with this.😇😇
